master/master.go

```diff
@@ -14,32 +15,32 @@ import (
 // mst defines the master struct, which is used to coordinate slaves and launch assembly, parsing, and
 // reporting slaves
 type mst struct {
-	filePath        string             // a path to a raw sequencing FASTQ file to perform assembly on
-	outPath         string             // a path to the location where results will be stored
-	numThreads      int                // number of threads to use for slave processes
-	assemblyResults chan result.Result // a collection of assembly results used by the assembly slave
-	parsingResults  chan result.Result // a collection of parsing results used by the parsing slave
+	filePath        string                // a path to a raw sequencing FASTQ file to perform assembly on
+	outPath         string                // a path to the location where results will be stored
```

Comment:
Just realized that `filePath` and `outPath` are on the config struct. So, they can also be removed. Similarly, the constructor `New` does not need them either 🤔